### PR TITLE
Feat/header toolbars

### PR DESCRIPTION
**Pull request by lustigy96**

### CODE REVIEW

File: src/common/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const { t } = useTranslation();
+  const getBlessing = (hour) => {
+    const morningHours = { morning: 5, afternoon: 12, evening: 17, night: 21 };
+    if (hour >= morningHours.morning && hour < morningHours.afternoon) {
+      return t('headerBlessings.morning');
+    }
+    if (hour >= morningHours.afternoon && hour < morningHours.evening) {
+      return t('headerBlessings.afternoon');
+    }
+    if (hour >= morningHours.evning && hour < morningHours.night) {
```
Comment (carmeltsadok) [inline src/common/Header/Header.jsx:22]:
```suggestion
    if (hour >= morningHours.evening && hour < morningHours.night) {
```

File: src/common/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { Toolbar, AppBar, IconButton, Button } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import useStyles from './Header.styles';
+
+const Header = () => {
+  const classes = useStyles();
+  const date = new Date();
+  const { t } = useTranslation();
+  const getBlessing = (hour) => {
+    const morningHours = { morning: 5, afternoon: 12, evening: 17, night: 21 };
```
Comment (carmeltsadok) [inline src/common/Header/Header.jsx:15]:
```suggestion
    const hoursRange = { morning: 5, afternoon: 12, evening: 17, night: 21 };
```

File: src/common/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    if (hour >= morningHours.morning && hour < morningHours.afternoon) {
+      return t('headerBlessings.morning');
+    }
+    if (hour >= morningHours.afternoon && hour < morningHours.evening) {
+      return t('headerBlessings.afternoon');
+    }
+    if (hour >= morningHours.evning && hour < morningHours.night) {
+      return t('headerBlessings.evning');
```
Comment (carmeltsadok) [inline src/common/Header/Header.jsx:23]:
```suggestion
      return t('headerBlessings.evening');
```

File: src/common/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    return t('headerBlessings.night');
+  };
+  return (
+    <AppBar position='static' className={classes.root}>
+      <Toolbar className={classes.toolbar}>
+        <Link to='/'>
+          <Button>
+            <img src='radar-logo.png' alt='' />
```
Comment (carmeltsadok) [inline src/common/Header/Header.jsx:32]:
```suggestion
            <img src={logo} alt='' />
```

File: src/common/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+// import MenuIcon from '@material-ui/icons/Menu';
```
Comment (carmeltsadok) [inline src/common/Header/Header.jsx:2]:
```suggestion
import logo from '../../utils/images/logo.svg';
```

File: src/common/SideBarNav/SideBarNav.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+import { Link } from 'react-router-dom';
+// import { Toolbar, AppBar, IconButton, Button } from '@material-ui/core';
+import DashboardCard from '../DashboardCard/DashboardCard';
+import useStyles from './SideBarNav.styles';
+import settingsIcon from '../../utils/images/aside/settings.svg';
+// import settingsIconActive from '../../utils/images/aside/settings-active.svg';
+import interviewIcon from '../../utils/images/aside/aside-interview-button.svg';
+// import interviewIconActive from '../../utils/images/aside/aside-interview-button-active.svg';
+import luzIcon from '../../utils/images/aside/aside-luz-button.svg';
+// import luzIconActive from '../../utils/images/aside/aside-luz-button-active.svg';
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.jsx:11]:
```suggestion
import React from 'react';
import { Link } from 'react-router-dom';
import DashboardCard from '../DashboardCard/DashboardCard';
import useStyles from './SideBarNav.styles';
import settingsIcon from '../../utils/images/aside/settings.svg';
import interviewIcon from '../../utils/images/aside/aside-interview-button.svg';
import luzIcon from '../../utils/images/aside/aside-luz-button.svg';
```

File: src/common/SideBarNav/SideBarNav.styles.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import { makeStyles } from '@material-ui/core/styles';
+// import apptheme from '../../theme';
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.styles.js:2]:
```suggestion
```

File: src/pages/XmlUpload/index.jsx [hunk 8]
```diff
@@ -8,12 +8,12 @@ const UploadXmlPage = () => {
   const classes = useStyles();
   const [files, setFiles] = useState([]);
   return (
-    <div className={classes.root}>
-      <DashboardCard className={classes.content}>
-        <DropZone files={files} setFiles={setFiles} />
-        <ProgressBoxes files={files} setFiles={setFiles} />
-      </DashboardCard>
-    </div>
+  // <div className={classes.root}>
```
Comment (carmeltsadok) [inline src/pages/XmlUpload/index.jsx:11]:
```suggestion
```

File: src/pages/XmlUpload/index.jsx [hunk 9]
```diff
@@ -8,12 +8,12 @@ const UploadXmlPage = () => {
...
+    <DashboardCard className={classes.content}>
+      <DropZone files={files} setFiles={setFiles} />
+      <ProgressBoxes files={files} setFiles={setFiles} />
+    </DashboardCard>
+  // </div>
```
Comment (carmeltsadok) [inline src/pages/XmlUpload/index.jsx:16]:
```suggestion
```

File: src/pages/XmlUpload/index.styles.js [hunk 10]
```diff
@@ -1,20 +1,19 @@
...
-  root: {
-    width: '100%',
-    textAlign: 'center',
-    alignItems: 'center',
-  },
+  // root: {
+  //   width: '100%',
+  //   textAlign: 'center',
+  //   alignItems: 'center',
+  // },
```
Comment (carmeltsadok) [inline src/pages/XmlUpload/index.styles.js:8]:
```suggestion
```

File: src/common/Header/Header.styles.js [hunk 11]
```diff
@@ -9,47 +9,30 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     height: '100%',
     justifyContent: 'space-between',
+    alignItems: 'center',
+    fontSize: '1.1rem',
+  },
+  blessing: {
+    marginRight: '0.5rem',
```
Comment (carmeltsadok) [inline src/common/Header/Header.styles.js:16]:
```suggestion
    fontFamily: theme.typography.fontFamily,
    marginRight: '0.5rem',
```

File: src/pages/XmlUpload/index.jsx [hunk 12]
```diff
@@ -8,12 +8,12 @@ const UploadXmlPage = () => {
...
   const [files, setFiles] = useState([]);
   return (
-    <div className={classes.root}>
-      <DashboardCard className={classes.content}>
-        <DropZone files={files} setFiles={setFiles} />
-        <ProgressBoxes files={files} setFiles={setFiles} />
-      </DashboardCard>
-    </div>
+  // <div className={classes.root}>
+    <DashboardCard className={classes.content}>
```
Comment (carmeltsadok) [inline src/pages/XmlUpload/index.jsx:12]:
```suggestion
    <DashboardCard className={classes.root}>
```

File: src/App.jsx [hunk 13]
```diff
@@ -2,10 +2,11 @@ import React, { useState, useCallback, useEffect } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import { ToastContainer } from 'react-toastify';
 import Home from './pages/Home/index';
+import Sidebar from './common/SideBarNav/SideBarNav';
 import AuthService from './services/auth.service';
 import UploadXmlPage from './pages/XmlUpload/index';
 import ConfigService from './services/config.service';
-import Header from './common/InterviewerHeader/InterviewerHeader';
+import Header from './common/Header/Header';
 import useStyles from './App.styles';
```
Comment (carmeltsadok) [inline src/App.jsx:10]: There is a need to add to class `bodyContainer` in `App.styles.js` file the next fields: 
`paddingTop: '2rem',
 height: 'calc(100vh - 9rem)',`

Comment (carmeltsadok) [reply src/App.jsx:10]: And to the `Home` page's `index.styles.js` file in `root` the field `width: '100%'` so we can see the home page in its full width.

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: mockserver/config.js [hunk 14]
```diff
@@ -1,3 +1,12 @@
+const sitesPostfixes = {
+  schedule: '/schedule',
+  track: '/track',
+  malshabSchedule: '/malshab-scheduling',
+  malshbSearch: '/search-malshab',
```
Comment (carmeltsadok) [inline mockserver/config.js:5]:
```suggestion
  malshabSearch: '/search-malshab',
```

File: src/common/SideBarNav/SideBarNav.styles.js [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'row',
+      width: '100%',
+      marginLeft: 'auto',
+      marginBottom: '1rem',
+      padding: '0.5rem 0',
+    },
+  },
+  icons: {
+    // marginTop: '2rem',
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.styles.js:22]:
```suggestion
```

File: src/common/SideBarNav/SideBarNav.jsx [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+    );
+  };
+  const iconsDetails = {
+    schedule: { urlPostfix: configApp.sitesPostfixes.schedule, imgDef: luzIcon, imgActive: luzIconActive, tooltip: 'schedule' },
+    track: { urlPostfix: configApp.sitesPostfixes.track, imgDef: interviewIcon, imgActive: interviewIconActive, tooltip: 'track' },
+    malshabScheduling: { urlPostfix: configApp.sitesPostfixes.malshabSchedule, imgDef: luzIcon, imgActive: luzIconActive, tooltip: 'schedule' },
+    searchMalshab: { urlPostfix: configApp.sitesPostfixes.malshbSearch, imgDef: luzIcon, imgActive: luzIconActive, tooltip: 'schedule' },
+    reports: { urlPostfix: configApp.sitesPostfixes.reports, imgDef: luzIcon, imgActive: luzIconActive, tooltip: 'schedule' },
+    posh: { urlPostfix: configApp.sitesPostfixes.posh, imgDef: luzIcon, imgActive: luzIconActive, tooltip: 'posh' },
+  };
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.jsx:35]: Move this to the declarations part.

File: src/common/SideBarNav/SideBarNav.jsx [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+
+const Sidebar = () => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const userRole = 'RAMAD_ITUR_OF_UNIT';
+  const createIcon = (iconDetails) => {
+    const src = iconDetails.urlPostfix === '/reports' ? iconDetails.imgActive : iconDetails.imgDef;
+    return (
+      <Link key={iconDetails.urlPostfix} className={classes.iconLink} to='/'>
+        <Tooltip placement='top' title={t(`xmlPage.${iconDetails.tooltip}`)}>
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.jsx:22]:
```suggestion
        <Tooltip placement='top' title={t(`tooltip.${iconDetails.tooltip}`)}>
```

File: src/common/SideBarNav/SideBarNav.jsx [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+import interviewIcon from '../../utils/images/aside/aside-interview-button.svg';
+import interviewIconActive from '../../utils/images/aside/aside-interview-button-active.svg';
+import luzIcon from '../../utils/images/aside/aside-luz-button.svg';
+import luzIconActive from '../../utils/images/aside/aside-luz-button-active.svg';
+import configApp from '../../appConf';
+
+const Sidebar = () => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const userRole = 'RAMAD_ITUR_OF_UNIT';
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.jsx:17]: Get user's role from `userStore`'s `userProfile`.

File: src/common/SideBarNav/SideBarNav.jsx [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+  const createIcon = (iconDetails) => {
+    const src = iconDetails.urlPostfix === '/reports' ? iconDetails.imgActive : iconDetails.imgDef;
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.jsx:19]: Why?

File: src/common/SideBarNav/SideBarNav.jsx [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+    return (
+      <Link key={iconDetails.urlPostfix} className={classes.iconLink} to='/'>
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.jsx:21]:
```suggestion
      <Link key={iconDetails.urlPostfix} className={classes.iconLink} to={iconDetails.urlPostfix}>
```

File: src/common/SideBarNav/SideBarNav.jsx [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+  };
+  const iconsToShow = Object.values(iconsDetails).filter(
+    (iconDetail) => configApp.allowedUrlPostfixesOfRole[userRole].includes(iconDetail.urlPostfix),
+  );
+  return (
+    <DashboardCard className={classes.root}>
+      <div className={classes.iconGroup}>
+        {iconsToShow.map((iconDetail) => createIcon(iconDetail))}
+      </div>
+      <Link className={classes.iconLink} to='/'>
```
Comment (carmeltsadok) [inline src/common/SideBarNav/SideBarNav.jsx:44]:
```suggestion
      <Link className={classes.iconLink} to='/settings'>
```

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/common/SideBarNav/SideBarNav.jsx (hunk 17 above)
Comment (lustigy96) [reply src/common/SideBarNav/SideBarNav.jsx:22]: tooltip is not so good, so I changes it to sidebar (my mistake :D )

File: src/common/SideBarNav/SideBarNav.jsx (hunk 19 above)
Comment (lustigy96) [reply src/common/SideBarNav/SideBarNav.jsx:19]: just as example... suppose to get the url page.. but the pages are not done and I wanted to check and see the difference 
When some pages will be on dev, I will change :)

File: src/common/SideBarNav/SideBarNav.jsx (hunk 21 above)
Comment (lustigy96) [reply src/common/SideBarNav/SideBarNav.jsx:44]: there is no page of settings... It is just an example.. 
When pages will be on dev I will synchronize with it..

File: src/common/SideBarNav/SideBarNav.jsx (hunk 19 above)
Comment (carmeltsadok) [reply src/common/SideBarNav/SideBarNav.jsx]: We should also add to the switch in `app.jsx` the routes from the config. We can add an array of objects. Each object can be like that: `{ link: '', page: <></>}`

**Review by carmeltsadok — APPROVED** [review]

**Pull request merged**